src/routes.php

```diff
@@ -100,9 +100,9 @@ public function generateTokenRequestUrl()
         $alias = $this->member->getFirstAlias();
         $tokenBuilder = \Tokenio\Http\Request\AccessTokenBuilder::createWithAlias($alias)->forAll();
 
-        $request = TokenRequest::builder($tokenBuilder->build())
-            ->addOption(TokenRequestOptions::REDIRECT_URL, 'http://localhost:3000/fetch-balances')
-            ->build();
+        $request = new TokenRequest();
```

Comment:
I don't think it was using the wrong class, we should use sdk level wrapper class here.